login/routers/SignIn.js

```diff
@@ -2,11 +2,10 @@ const express = require('express');
 
 const router = express.Router();
 const LoginRequest = require('../db/db_objects/LoginRequest.js');
-const signInValidation = require('../core/validations/SignInValidation.js');
+const UserResponseServices = require('./controllers/UserResponseServices.js');
 
-
-router.post('/', (request, response) => {
+router.post('/', (request, response,next) => {
   const loginRequest = new LoginRequest(request.body.login, request.body.password);
-  signInValidation.SignInValidation(loginRequest, response);
+  UserResponseServices.SignIn(loginRequest, response);
```

Comment:
Use singular names consistently (Controller, Repository, Service, Validator). Also, user response services doesn't really make sense as a name. Maybe name the class UserService and put cohesive functionality in there?

Reply:
Create UserService that has functions like UserExists => bool, CreateUser => void and AuthentificationService that has functions like SignIn and SignUp and Logout